solutions_template/cli/set.py

```diff
@@ -16,15 +16,18 @@
 
 import typer
 import glob
-from pathlib import Path
+import pathlib
 from typing import Optional
 from typing_extensions import Annotated
 from copier import run_auto
 from .cli_utils import *
 
 set_app = typer.Typer()
 
-FILE_EXTENSIONS = ["yaml", "env", "tfvars", "sh", "md"]
+INCLUDE_PATTERNS = [
+    "*.yaml", "*.yml", "*.env", "*.tfvars", "*.tf", "*.sh", "*.md"
+]
+EXCLUDE_PATTERNS = ["**/.terraform/**/*.*", "**/node_modules/**/*.*"]
```

Comment:
We should also exclude terraform.tfstate and terraform.tfstate.backup

Reply:
Intentionally not to change the state directly. This command only updates from the raw clone from this repo, to start with a new deployment with new terraform runs.